sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLogRecordData.java

```diff
@@ -12,20 +12,22 @@
 import io.opentelemetry.api.trace.SpanContext;
 import io.opentelemetry.sdk.common.InstrumentationScopeInfo;
 import io.opentelemetry.sdk.logs.data.LogRecordData;
+import io.opentelemetry.sdk.logs.data.internal.ExtendedLogRecordData;
 import io.opentelemetry.sdk.resources.Resource;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 
 @AutoValue
 @AutoValue.CopyAnnotations
 @Immutable
-abstract class SdkLogRecordData implements LogRecordData {
+abstract class SdkLogRecordData implements LogRecordData, ExtendedLogRecordData {
```

Comment:
Is it necessary to implement both?
```suggestion
abstract class SdkLogRecordData implements ExtendedLogRecordData {
```